app/models/product_drive.rb

```diff
@@ -44,7 +45,7 @@ def end_date_is_bigger_of_end_date
     return if start_date.nil? || end_date.nil?
 
     if end_date < start_date
-      errors.add(:end_date, 'End date must be after the start date')
```

Comment:
Fixes this duplicate text issue:

![Screenshot from 2025-01-07 10-16-28](https://github.com/user-attachments/assets/1826bab5-d3db-444c-8106-5f4e3372cf71)

Reply:
Unrelated change, so I can move it to another PR if needed.